test/test.js

```diff
@@ -478,6 +478,26 @@ describe('removeHeaders', function() {
   });
 });
 
+describe('setHeaders', function() {
+  before(function() {
+    cors_anywhere = createServer({
+      setHeaders: {'x-powered-by': 'CORS Anywhere'},
+    });
+    cors_anywhere_port = cors_anywhere.listen(0).address().port;
+  });
+  after(stopServer);
+
+  it('GET /example.com', function(done) {
+    request(cors_anywhere)
+      .get('/example.com/echoheaders')
+      .expect('Access-Control-Allow-Origin', '*')
+      .expectJSON({
+        host: 'example.com',
+        'x-powered-by': 'CORS Anywhere'
+      }, done);
+  });
```

Comment:
Note for myself: after accepting the PR, add another test where `setHeaders` is combined with `removeHeaders`, to make sure that setHeaders will continue to work as intended in the future when the same header is in both lists.